tools/regenerate_docs.py

```diff
@@ -131,7 +131,8 @@ def generate_wrapdb_table(output_dir: Path) -> None:
             dependency_names_str = ', '.join(dependency_names)
             program_names = info.get('program_names', [])
             program_names_str = ', '.join(program_names)
-            f.write(f'| {name} | {versions_str} | {dependency_names_str} | {program_names_str} |\n')
+            website = info.get('website', '#')
+            f.write(f'| [{name}]({website}) | {versions_str} | {dependency_names_str} | {program_names_str} |\n')
```

Comment:
Markdown links should only be generated if there is an actual website to link to. Otherwise, dummy links are generated which are visually indistinguishable from *actual* links.

With the current setup, hotdoc also complains about a lot of `Trying to link to non-existing identifier ‘#’`.

Reply:
Regardless if there is no website defined the entry should be empty, otherwise something like `[website](https://url.to.upstream)`.

Reply:
Should be done. Is there any particular place where the JSON schema is documented? I believe it should be mentioned that the "website" field is optional.

Reply:
@paulherman the json format is loosely defined in wrapdb's README. But again, I think it makes sense to have that info in the .wrap file instead of releases.json, it can be generally useful even outside of wrapdb.

Reply:
That makes sense. I think this PR stands alone since passing this through the .wrap file is in a different repo and I'll do the following:
1. change meson/mesonbuild/wrap/wraptool.py, specifically `info` to parse the website
2. change wrapdb/tools/import-wraps.py to extract the website from the `meson wrap info <name>` command

I think these can also be individual PRs. WDYT @xclaesse?

Reply:
You don't need to change `wrapdb/tools/import-wraps.py` that's a script that I used for the initial import into wrapdb, it's not used any more. Could delete it actually.

Hmm, actually I did not realize that we don't have the wrap file when generating this doc page. Downloading each wrap file from wrapdb individually is not going to scale... You're probably right that we need that website into releases.json then.